plugins/GSdx/res/convert.fx

```diff
@@ -389,6 +395,21 @@ PS_OUTPUT ps_main17(PS_INPUT input)
 	return output;
 }
 
+// DUMMY
+PS_OUTPUT ps_main18(PS_INPUT input)
+{
+	PS_OUTPUT output;
+	output.c = input.p;
+	return output;
+}
+
+PS_OUTPUT ps_main19(PS_INPUT input)
+{
+    PS_OUTPUT output;
+    output.c = input.c * float4(1.0, 1.0, 1.0, sample_c(input.t).r);
+    return output;
```

Comment:
Tabs instead of spaces, also small nitpick but I prefer if there is an empty space between each line, same goes for above to keep things consistent.
```
PS_OUTPUT ps_main18(PS_INPUT input)
{
	PS_OUTPUT output;

	output.c = input.p;

	return output;
}
```